### ISSUE TITLE
Code component theming

### ISSUE THREAD

**Issue by irgolic** [gradio-app/gradio#6345]
- [X] I have searched to see if a similar issue already exists.

**Is your feature request related to a problem? Please describe.**  
There's no way to override a Code component's codemirror theme.

**Describe the solution you'd like**  
Ideally a kwarg on the Code component taking a string that's the codemirror theme name.

**Additional context**  
[Link to corresponding Discord thread](https://discord.com/channels/879548962464493619/1171579673675837450/1171579673675837450)

**Comment by pngwn** [comment]
Thanks for filing this issue after our discussion on Discord @irgolic! We'll take a look at this as soon as we can.

**Comment by abidlabs** [comment]
I don't think this makes sense to include in the `Code` component as it may produce a component that is styled completely separately from the rest of the Gradio app. If this is really needed, then I'd recommend making a custom component!

If you'd like to create your own (you can template off of the existing `Code` component), we've put together a Guide: https://www.gradio.app/guides/five-minute-guide, and we're happy to help. I

**Comment by pngwn** [comment]
Not sure about this. Low prio for sure but it is common for people to have different preferences for syntax themes (which is mostly what this feature is about). 

This would grant the ability to themes the whole of the editor component independently.

**Comment by abidlabs** [comment]
Oh sorry I misunderstood, I didn't realize it was referring to syntax highlighting

**Comment by abidlabs** [comment]
Thanks for the suggestion @irgolic! Code component theming is something users can already largely achieve via custom CSS / themes, and a full CodeMirror theme-name API isn't something we're planning to prioritize.
